api/management/swagger-api_management.yaml

```diff
@@ -1343,8 +1343,8 @@ paths:
             application/json:
               schema:
                   $ref: '#/components/schemas/AuthorizationResponse'
-        400:
-          description: invalid information provided
+        404:
+          description: Not found
```

Comment:
not found (it seems only "summary" values starts with an uppercase... let's go on like this)

Reply:
Good catch ! Corrected in commit e241ce2e684e158e8f8acfa3647de8ea127494e1